src/main/io/osd.c

```diff
@@ -158,7 +158,7 @@ static displayPort_t *osdDisplayPort;
 #define AH_SIDEBAR_WIDTH_POS 7
 #define AH_SIDEBAR_HEIGHT_POS 3
 
-PG_REGISTER_WITH_RESET_FN(osdConfig_t, osdConfig, PG_OSD_CONFIG, 1);
+PG_REGISTER_WITH_RESET_FN(osdConfig_t, osdConfig, PG_OSD_CONFIG, 2);
```

Comment:
No need to increase the version here, since you didn't touch `osdConfig_t`, just exposed its already existing fields. Doesn't matter because this is going to be split into 2 groups (elements and settings) before 2.0 and we'll need to increase the version anyway, so no harm in increasing it in this PR.